libpod/runtime.go

```diff
@@ -210,17 +210,6 @@ func newRuntimeFromConfig(ctx context.Context, conf *config.Config, options ...R
 		}
 	}
 
-	if err := shutdown.Register("libpod", func(sig os.Signal) error {
-		// For `systemctl stop podman.service` support, exit code should be 0
-		if sig == syscall.SIGTERM {
-			os.Exit(0)
-		}
-		os.Exit(1)
-		return nil
-	}); err != nil && !errors.Is(err, shutdown.ErrHandlerExists) {
-		logrus.Errorf("Registering shutdown handler for libpod: %v", err)
-	}
```

Comment:
Doesn't this make podman ignore all SIGTERM ,SIGINT signals and never exit? I don't see how this can ever work without it.

Reply:
yeah looks like we need it.  Replaced with a different one